dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OrganizationIT.java

```diff
@@ -1241,12 +1241,11 @@ public void testBasicCollections() {
         containersApi.publish(entryId, publishRequest);
 
         // Add tool to collection
-        organizationsApi.addEntryToCollection(organization.getId(), collectionId, entryId);
+        organizationsApi.addEntryToCollection(organization.getId(), collectionId, entryId, null);
 
         // There should be two entries for collection with ID 1
-        final long count2 = testingPostgres
-            .runSelectStatement("select count(*) from collection_entry where collectionid = '1'", long.class);
-        assertEquals("There should be 2 entries associated with the collection, there are " + count2, 2, count2);
+        Collection collectionById = organizationsApi.getCollectionById(organizationID, collectionId);
+        Assert.assertEquals(2, collectionById.getEntries().size());
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'Assert.assertEquals' due to existing static import 'org.junit.Assert.assertEquals'](https://app.codacy.com/gh/dockstore/dockstore/pullRequest?prid=6286359)